src/Facilitator/index.js

```diff
@@ -639,6 +651,12 @@ class Facilitator {
       );
       return Promise.reject(err);
     }
+    const latestAnchorInfo = await this.coGateway.getLatestAnchorInfo();
+    if ((new BN(latestAnchorInfo.blockHeight)).lt(new BN(blockNumber))) {
+      logger.error('Latest available state root on target is lower than the height at which stake was done!');
+      const err = new TypeError(`Latest available state root on target is lower than the height at which stake was done!`);
```

Comment:
This should be `Error` instead of `TypeError`.

The error message is not clear, its comparing state root with height. 
Eg: Block number should not be greater than the latest available state root block height.

Reply:
Done!